index.js

```diff
@@ -17,6 +17,12 @@ var FindParent = {
     return this.byMatcher(element, function(el) {
       return el.classList.contains(className);
     });
+  },
+
+  byAttribute: function(element, attName) {
+    return this.byMatcher(element, function(el) {
+      return el.hasAttribute(attName);
```

Comment:
But if the goal of this PR is to find an element with a data attribute, you probably want to just check if something is on the `dataset` instead of using `hasAttribute`